src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersStatusByEmailTest.java

```diff
@@ -48,17 +51,20 @@ public void ac2_find_user_status_by_email_with_pui_case_manager_role_should_retu
         userRoles.add("pui-case-manager");
         // creating new user request
         NewUserCreationRequest userCreationRequest = createUserRequest(userRoles);
-        // creating user in idam with the same email used in the invite user so that status automatically will be update in the up
-        professionalApiClient.getMultipleAuthHeadersExternal(puiUserManager, userCreationRequest.getFirstName(), userCreationRequest.getLastName(), userCreationRequest.getEmail());
+        // creating user in idam with the same email used in the invite user so that status automatically will be
+        // update in the up

```

Comment:
This comment would read better if it was split earlier, suggestion:

```
        // creating user in idam with the same email used in the invite user
        // so that status automatically will update in the up
```